crates/bevy_ui/src/ui_node.rs

```diff
@@ -289,46 +458,123 @@ pub struct Style {
+
     /// The amount of space between the margins of a node and its padding.
     ///
     /// If a percentage value is used, the percentage is calculated based on the width of the parent node.
     ///
     /// The size of the node will be expanded if there are constraints that prevent the layout algorithm from placing the border within the existing node boundary.
     ///
     /// Rendering for borders is not yet implemented.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/border-width>
+    ///
+    /// &nbsp;
     pub border: UiRect,
-    /// Defines how much a flexbox item should grow if there's space available
+
+    /// Whether a Flexbox container should be a row or a column. This property has no effect of Grid nodes.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction>
+    ///
+    /// &nbsp;
+    pub flex_direction: FlexDirection,
+
+    /// Whether a Flexbox container should wrap it's contents onto multiple line wrap if they overflow. This property has no effect of Grid nodes.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap>
+    ///
+    /// &nbsp;
+    pub flex_wrap: FlexWrap,
+
+    /// Defines how much a flexbox item should grow if there's space available. Defaults to 0 (don't grow at all).
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow>
+    ///
+    /// &nbsp;
     pub flex_grow: f32,
-    /// How to shrink if there's not enough space available
+
+    /// Defines how much a flexbox item should shrink if there's not enough space available. Defaults to 1.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink>
+    ///
+    /// &nbsp;
     pub flex_shrink: f32,
-    /// The initial length of the main axis, before other properties are applied.
+
+    /// The initial length of a flexbox in the main axis, before flex growing/shrinking properties are applied.
     ///
-    /// If both are set, `flex_basis` overrides `size` on the main axis but it obeys the bounds defined by `min_size` and `max_size`.
-    pub flex_basis: Val,
-    /// The ideal size of the flexbox
+    /// `flex_basis` overrides `size` on the main axis if both are set,  but it obeys the bounds defined by `min_size` and `max_size`.
     ///
-    /// `size.width` is used when it is within the bounds defined by `min_size.width` and `max_size.width`.
-    /// `size.height` is used when it is within the bounds defined by `min_size.height` and `max_size.height`.
-    pub size: Size,
-    /// The minimum size of the flexbox
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis>
     ///
-    /// `min_size.width` is used if it is greater than either `size.width` or `max_size.width`, or both.
-    /// `min_size.height` is used if it is greater than either `size.height` or `max_size.height`, or both.
-    pub min_size: Size,
-    /// The maximum size of the flexbox
+    /// &nbsp;
+    pub flex_basis: Val,
```

Comment:
(afaik) `flex_basis` shouldn't have any effect with `Display::Grid` but at the moment it controls the length of the inline-axis.

A simple workaround in Bevy would be to map it to `Auto` if `Grid` is enabled? maybe you'd rather fix it in taffy though, I've no idea.

Reply:
> at the moment it controls the length of the inline-axis.

It does?? Do you have a reproduction for this? I'd definitely want to fix this in Taffy.